en/docs/references/postman-integration.md

```diff
@@ -0,0 +1,129 @@
+
+3. Once the service opens, click on the **Test** icon in the left pane. Then click on the **Postman** icon.
+
+    ![Access Postman test view](../assets/img/postman/access-postman-test-view.png){.cInlineImage-full}
+    
+    As shown in the image above, the **Postman** panel opens to the right of the low-code diagram with instructions to integrate Postman. You have already completed the first two steps given there.
+
+4. Click on the link provided in Step 3 in the **Postman** panel.
+
+5. In the **API Key** field, enter the API key that you generated in the Postman application.
+
+    Once you enter it, the following message appears below the field.
+    
+    ```text
+    Your API key was validated by Choreo
+    ```
+   
+6. Click **Save**. 
+
+Now you can proceed to create a new workspace to link to your service or select an existing workspace in your Postman account.
+
+## Step 3: Link a workspace to your service
+
+In this step, let's connect a workspace to your service by following this procedure:
+
+1. In the **Postman** right panel that you opened, expand the **Select or create a workspace** field.
+
+    ![Postman right panel](../assets/img/postman/postman-right-panel.png){.cInlineImage-full}
+    
+    As shown above, the drop-down list displays the existing workspaces in your Postman account. You can link your service to an existing workspace or create a new one. In this example, let's create a new workspace.
+    
+2. Click **+ Create New**.
+
+3. Enter a name and a description for the new workspace. For this example, let's enter them as follows:
+
+    | **Field**       | **Value**                       |
+    |-----------------|---------------------------------|
+    | **Name**        | `covid-Status`                  |
+    | **Description** | `Test the covid-Status service` |
+    
+4. Click **Create & Link**.
+
+Now you have linked your service to the new workspace named `covid-Status`.
+
+## Step 4: Create a collection
+
+!!! tip
+    If you linked an existing workspace to the service and that workspace has collections that are already created in it, you can skip this step.
+    
+To create a collection, follow this procedure:
+
+1. Under **Collections** in the **Postman** right panel, click **+ Add**.
+
+2. Enter a name and a description. For this example, let's enter them as follows:
+
+    | **Field**       | **Value**                             |
+    |-----------------|---------------------------------------|
+    | **Name**        | `collection1`                         |
+    | **Description** | `Collection for covid-status service` |
+    
+3. Click **Create**.
+
+As a result, the collection is created and displayed under **Collections** in the **Postman** right panel.
+
+## Step 5: Create a request
+
+To create a request, follow this procedure:
+
+1. In the **Postman** right panel, click on the collection that you created. This opens the **Requests** section.
```

Comment:
It is not on the collection, we need to say click on the link icon on the collection.

Reply:
Corrected.